### DISCUSSION TITLE
essay/2023/12/14/50000-coffee

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#152] [Comments]
# essay/2023/12/14/50000-coffee

사용자 10만 명의 서비스 만들기란 글을 쓴 적이 있습니다. 10만 명을 목표로 해보겠다고. 누적 가입자 수 이긴 하지만… 오늘 커피한잔의 가입자가 5만 명을 넘어섰습니다.

https://jeho.page/essay/2023/12/14/50000-coffee.html

**Comment by ANAHEIMS** [comment]
5만명..! 축하드립니다

**Reply by BenjaminKim** [reply]
축하 고맙습니다! 💚

**Comment by jinho6225** [comment]
글로벌 진출 계획은 없으신가요?? 펀딩 안받고 혼자서 계속 하실꺼라고 인터뷰에서 이야기 하신건 아는데 1인이 글로벌 진출도 가능하잖아요? 그래서 여쭤봅니다. 미국에 진출해고 좋을것 같아요(개인적인 생각이고 시장조사는 해보지 않았습니다)

**Reply by BenjaminKim** [reply]
내년쯤에는 한 번 해보려고 계획하고 있습니다. [관련글](https://jeho.page/essay/2023/09/29/give-back-more-than-take.html)
커피한잔으로 해볼수도 있고.. 다른 종류의 서비스로 해볼 수도 있을 것 같습니다.
커피한잔을 미국에 가져갈 생각은 해보지 못했는데 추천해주시는 이유가 있으신가요?(일본이나 동남아 정도는 괜찮지 않을까 생각해본 적은 있습니다)

**Reply by jinho6225** [reply]
단순하게 제가 미국에 살고 있는 개발자이고 제 주변에 데이팅앱으로 결혼한 사람들이 생각보다 많아서요 그리고 미국은 한국에 비해서는 데이팅앱에 대한 이미지가 나쁘지 않은 것 같아요 그리고 시장 규모도 엄청 크구요

**Reply by BenjaminKim** [reply]
그렇군요. 제가 미국 경험이 몇 주 정도 밖에 없어서 미국에서 사는 사람들의 모습을 잘 상상할 수 없거든요. 이야기 들려주셔서 감사합니다. 다 열어놓고 한 번 고려해보겠습니다. 😁

**Reply by jinho6225** [reply]
차한잔 할 수 있는 기회가 있으면 제가 벤자민님께 여러모로 많이 배울 수 있는 시간이 될 것 같습니다 👍

**Reply by BenjaminKim** [reply]
미국에 계신 것 아닌가요? 강남역 근처에 오시게 되면 연락주세요!

**Reply by jinho6225** [reply]
한국 방문중이예요ㅎ 한국에서 미국 시간으로 일하고 있습니다. 편하신날 알려주세요! 찾아뵙겠습니다!ㅎ

**Reply by BenjaminKim** [reply]
제 이메일로 연락주시겠어요? 블로그 footer 에 있습니다.

**Reply by synuns** [reply]
커피한잔 개발자와 커피한잔하기..!
저도 로망입니다!

**Reply by BenjaminKim** [reply]
@synuns [이 글](https://jeho.page/essay/2024/01/05/coffee-with-coffee-developer.html)을 참고해주세요..!
